hackathon_site/dashboard/frontend/src/slices/event/teamDetailSlice.ts

```diff
@@ -80,6 +120,26 @@ const teamDetailSlice = createSlice({
             state.isLoading = false;
             state.error = payload?.message ?? "Something went wrong";
         });
+        builder.addCase(updateParticipantIdProvided.pending, (state) => {
+            state.isLoading = true;
+            state.error = null;
```

Comment:
For the error state, technically you should create a new state for the checkbox error too, but I don't think it's necessary here, dispatching the snackbar is enough. So you can remove all the error setting.

Reply:
ok, error stuff removed